src/clr-base/tsconfig.spec.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "outDir": "../../out-tsc/spec",
+    "types": ["jasmine", "node"],
+    "target": "es2015"
```

Comment:
ES2015 is probably fine here unless we decide to run tests in an older environment.